doc/build-unix.md

```diff
@@ -288,26 +288,3 @@ This example lists the steps necessary to setup and build a command line only di
     ./src/bitcoind
```

Comment:
unrelated: Shouldn't the arch linux section be in the section with the other linux (Debian and Fedora)?